website/package.json

```diff
@@ -33,21 +32,17 @@
 	"devDependencies": {
 		"@types/classnames": "^2.3.1",
 		"@types/glob": "^8.1.0",
-		"@types/html-webpack-plugin": "^3.2.2",
 		"@types/react": "^17.0.3",
 		"@types/react-dom": "^17.0.3",
 		"@types/webpack": "^4.41.10",
 		"clean-webpack-plugin": "^3.0.0",
 		"copy-webpack-plugin": "^11.0.0",
 		"css-loader": "^3.5.1",
-		"file-loader": "^6.0.0",
```

Comment:
the file-loader is deprecated in Webpack.
Instead of the file-loader  must be used `asset/resource`.